bibtexparser/bibdatabase.py

```diff
@@ -66,6 +67,12 @@ def __init__(self):
         #: List of fields that should not be updated when resolving crossrefs
         self._not_updated_by_crossref = ['_FROM_CROSSREF']
 
+    def __eq__(self, other: Any) -> bool:
+        """Equality == method."""
```

Comment:
This docstring does not really provide a useful description.

State what's considered equal (e.g. same content, same order, ...)